tdrs-frontend/src/components/PrivateTemplate/PrivateTemplate.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+
+
+export default function PrivateTemplate({ title, children }) {
+
+    useEffect(() => {
+        document.title = `${title} - TDP - TANF Data Portal`
+    })
+
+    return (
+        <div className="grid-container margin-top-3" >
+            <h1 className="request-access-header font-serif-2xl">{title}</h1>
```

Comment:
There have been some changes to the `h1` tags for styling. The class `request-access-header` is no longer necessary and there are some bottom margin attributes that need to be updated. Also, when the timeout modal pops up, I'm not sure if it's necessary to change the title but, if so, that h1 has different styles than the normal page `h1`'s. I'm not sure if the best solution is to pass in the classes that we want attributed to the `h1`? I'm open to suggestions on that.

Reply:
@riatzukiza I just looked at the USA jobs site and they DO NOT update the title when the timeout modal pops up. So if you can just please make sure that the classes that are on `h1` elements on `raft-tdp-main` are the same as the classes in this template `h1` we should be good to go!